src/lib.rs

```diff
@@ -239,14 +220,14 @@ impl Gdbm {
 
     // API: export database to binary dump file
     pub fn export_bin(&mut self, outf: &mut std::fs::File, mode: ExportBinMode) -> io::Result<()> {
-        let is_lfs = match mode {
-            ExportBinMode::ExpNative => self.header.is_lfs,
-            ExportBinMode::Exp32 => false,
-            ExportBinMode::Exp64 => true,
+        let format = match mode {
+            ExportBinMode::ExpNative => self.header.format(),
+            ExportBinMode::Exp32 => Format::Standard,
+            ExportBinMode::Exp64 => Format::Large,
```

Comment:
English language nits:  "Format" appears to reference to large-file-size (LFS) feature specifically.

This confuses, because GDBM also has multiple file formats Old / New / Numsync.